node_addons/FormatConverter/setenv.sh

```diff
@@ -1,118 +1,149 @@
+	    confCmd="./configure --disable-everything --disable-doc --enable-protocol=file --enable-demuxer=mpegts --enable-muxer=rtp_mpegts --enable-parser=h264 --enable-parser=aac --enable-muxer=mp4 --enable-zlib --enable-bsf=aac_adtstoasc --enable-decoder=aac --enable-decoder=h264 --enable-muxer=flv --enable-protocol=rtmp --enable-encoder=libmp3lame $debug_specifics"
 
+	    [ "$os_name" == "Linux" ] && confCmd="$confCmd --enable-pic"
 
-    [ "$os_name" == "Linux" ] && confCmd="$confCmd --enable-pic"
+	    actualCmd=""
 
-    actualCmd=""
+	    [ -f "$configFileName" ] && actualCmd=`cat $configFileName`
 
-    [ -f "$configFileName" ] && actualCmd=`cat $configFileName`
+	    echo -e "actualCmd=\n<$actualCmd>"
+	    echo -e "confCmd=\n<$confCmd>"
+	    if [ "$actualCmd" != "$confCmd" ]
+	    then
+	        echo "configuring ffmpeg..."
+	        eval "$confCmd"
+	    else
+	        echo "no need to run configure"
+	    fi
 
-    echo -e "actualCmd=\n<$actualCmd>"
-    echo -e "confCmd=\n<$confCmd>"
-    if [ "$actualCmd" != "$confCmd" ]
-    then
-        echo "configuring ffmpeg..."
-         eval "$confCmd"
-    else
-        echo "no need to run configure"
-    fi
-
-    echo "$confCmd" > $configFileName
+	    echo "$confCmd" > $configFileName
 
-    make &> /dev/null
+	    make &> /dev/null
+	popd
 
 }
 
-path=`dirname ${BASH_SOURCE[0]}`
+echo "current path `pwd`"
 
 `which node-gyp` || npm install node-gyp -g
 
 [ -d '/usr/local/lib/node_modules/nan' ] || npm install nan -g
 
-cd $path
-
-path=`pwd`
-
-[ -d "$path/build" ] || mkdir -p "$path/build"
-
-makeFFmpeg $path/build
-
-cd $path
+makeFFmpeg
 
 gyp_args=''
 
 echo "Installing NAN"
 npm install nan
 
-case $os_name in
+case `uname` in
 'Darwin')
     echo "Mac OS"
     gyp_args='-- -f xcode'
     echo "$gyp_args"
     node-gyp configure $gyp_args
+    FORMAT_CONVERTER_BIN=FormatConverter.dylib
     ;;
 *) ;;
 esac
 
 echo "Start node-gyp configure"
 node-gyp configure
 
-if [ "$build_conf" = "Debug" ]; then
+debugExt=''
+
+if [ "$BUILD_CONF" = "Debug" ]; then
     gyp_debug="--debug"
     debugExt=".debug"
 fi
 echo "Start node-gyp build. $gyp_debug"
 node-gyp build $gyp_debug -v
 
-case $os_name in
- 'Linux')
-    cp `pwd`/build/$build_conf/FormatConverter.so "`pwd`/../../bin/linux/FormatConverter.node$debugExt"
-    ;;
-'Darwin')
-    cp `pwd`/build/$build_conf/FormatConverter.dylib "`pwd`/../../bin/darwin/FormatConverter.node$debugExt"
-    ;;
-*) ;;
-esac
\ No newline at end of file
+echo "cp ${FFMPEG_BUILD_PATH}/${FORMAT_CONVERTER_BIN} ${BIN_PATH}/FormatConverter.node$debugExt"
+
+cp "${FFMPEG_BUILD_PATH}/${FORMAT_CONVERTER_BIN}" "${BIN_PATH}/FormatConverter.node$debugExt"
```

Comment:
1.please delete the FormatConverter.node from git
2.you didn't fix the actual javascript code that loads the FormatConverter.node with the 'new' path